### PR TITLE
#160583374 Fix responses for content not found and permission required

### PR DESCRIPTION
**Pull request by ssewilliam**
### What does this PR do?
- Adds custom layouts for the page not found and login required

### How should this be manually tested?
- When reading content while not logged in Application does not redirect to home.
- Accessing a wrong link also does not redirect to the home page

### Any background context you want to provide?
- I took advantage of the already implemented redux on the home page.

### What are the relevant pivotal tracker stories?
- [#160583374](https://www.pivotaltracker.com/story/show/160583374)

### coverage screenshot
screen shot 2018-09-18 at 15 11 13

### CODE REVIEW

File: src/CSS/Errors.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+.mainErrorText {
+  font-size: 60px;
+  font-family: 'Mallanna',
+    sans-serif;
+}
+
+.btn_color {
+  color: #fff !important;
+}
```
Comment (lytes20) [inline src/CSS/Errors.css]: end of line missing

Comment (ssewilliam) [reply src/CSS/Errors.css]: @lytes20 
This was worked on

**Comment by geofrocker** [comment]
The footer should keep at the bottom of the page

**Review by AGMETEOR — APPROVED** [review]

**Comment by AGMETEOR** [comment]
@ssewilliam please make sure that when there are less than 5 articles, they still show on the infinite scroll

**Comment by ssewilliam** [comment]
@geofrocker,
Should what @AGMETEOR is suggesting be included in this PR as well?

**Comment by geofrocker** [comment]
I don't get @AGMETEOR's suggestion clearly. Can you have a sync with him?

**Review by phillipseryazi — APPROVED** [review]

**Review by lytes20 — APPROVED** [review]

**Pull request merged**